docs: Add documentation for `spice chat` CLI command (#962)
spiceai/docs: website/docs/cli/reference/chat.md

> ---

Reviewer: Needs description.

title: 'chat'
sidebar_label: 'chat'
pagination_prev: null
pagination_next: null
---

> Start an interactive or one-shot chat with any [model](/docs/components/models/index.md) currently ready in the Spice runtime.

Reviewer: ```suggestion
Start an interactive or one-shot chat with a [model](/docs/components/models/index.md) registered in the Spice runtime.
```

> ### Requirements

Reviewer: Wrong MD indents.

---
title: 'chat'
sidebar_label: 'chat'
pagination_prev: null
pagination_next: null
description: 'spice chat CLI documentation'
---

Start an interactive or one-shot chat with a [model](/docs/components/models/index.md) registered in the Spice runtime.

## Requirements

- Spice runtime must be running
- At least one model defined in `spicepod.yaml` and the model is ready

## Usage

### Interative Chat: Invoke the command without arguments to open a REPL

```shell
spice chat [flags]
```

### One-shot Chat: Pass a single message as the argument to send a one-shot chat request and print the response

```shell

> spice chat [flags] <message>

Reviewer 2: since message is optional argument

```suggestion
spice chat [flags] [<message>]
```
